crates/graphql/src/schema/objects/nft.rs

```diff
@@ -352,67 +409,68 @@ impl TryFrom<models::Nft> for Nft {
     }
 }
 
-#[graphql_object(Context = AppContext)]
 impl Nft {
-    pub fn address(&self) -> &str {
+    fn _uri(&self) -> &str {
+        &self.uri
+    }
+
+    fn _slot(&self) -> Option<i32> {
+        self.slot
+    }
+
+    fn _model(&self) -> Option<String> {
+        self.model.as_ref().map(ToString::to_string)
+    }
+
+    fn _address(&self) -> &str {
         &self.address
     }
 
-    pub fn name(&self) -> &str {
+    fn _name(&self) -> &str {
```

Comment:
Why are all of these _ now?

Reply:
Two issues:

1. I ended up having to do 4 `impl` of the same methods and wanted a single "source" implementation to refer to. Compounds with
2. Rust wasnt able to determine that the `impl Nft` block without underscore method names was the one I was referring to when doing e.g. `self.name()` or `self.0.name()` in the `impl NftExt` blocks (since "self" in those blocks refers to `NftExt` rather than the struct for which the trait is being implemented). This led to infinite recursion.

This was my workaround :/